### ISSUE TITLE
how to output aggregate result from a view to postgres table

### ISSUE THREAD

**Issue by 383747787** [pipelinedb/pipelinedb#2021]
i want to save the aggregate result from a view  to one postgres table
for example
CREATE CONTINUOUS VIEW v1 as
select sum(amount) as sum_amount,count(*) as all_count from some_steam

and then 
i want to save the aggregate result to one table  for a long time
can pipelinedb support this feature？

**Comment by derekjn** [comment]
> can pipelinedb support this feature？

@383747787 yes! You just need to use a transform with an [output function](http://docs.pipelinedb.com/continuous-transforms.html#creating-your-own-output-function) that writes rows out to the table of your choice.

So in your case, you'd have `v1`:

```sql
CREATE VIEW v1 AS
 SELECT sum(amount) AS sum_amount, count(*) AS all_count
FROM some_steam;
```

And then a [transform](http://docs.pipelinedb.com/continuous-transforms.html) that reads that CV's output stream and calls an output function that writes out to a table:

```sql
CREATE VIEW xform WITH (action=transform, outputfunc=insert_into_table) AS
  SELECT (new).sum_amount AS sum_amount, (new).count AS all_count
FROM output_of('v1')
```
